codemeta.json

```diff
@@ -253,31 +253,18 @@
         }
     ],
     "codeRepository": "https://github.com/gammapy/gammapy",
-    "dateCreated": "2013-08-19",
-    "datePublished": "2014-08-25",
-    "dateModified": "2024-02-29",
```

Comment:
This should be kept!

Reply:
I have no idea why this file was modified ...
Should we exclude the modif in this file completly from the PR ?

Reply:
An automated script was applied to the PR. It is normally triggered when the CITATION.cff is modified. I don't know why it was triggered here. You can remove the file safely.

Reply:
Even `dateCreated` and `datePublished` which is the first version of Gammapy.
`dateModified` is the latest version